<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactBuilder</class>
 <widget class="QDialog" name="ContactBuilder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="0">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="ForenameLabel">
      <property name="text">
       <string>Forename: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="SurnameLabel">
      <property name="text">
       <string>Surname: </string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="EMailLabel">
      <property name="text">
       <string>E-Mail Address: </string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="TwitterHandleLabel">
      <property name="text">
       <string>Twitter Username: </string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="SkypeHandleLabel">
      <property name="text">
       <string>Skype Username: </string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="LastFmHandleLabel">
      <property name="text">
       <string>Last.FM Username: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="Forename"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="Surname"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="EMail"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="TwitterHandle"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="SkypeHandle"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="LastFmHandle"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="TickerColourLabel">
      <property name="text">
       <string>Ticker Item Colour: </string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="SelectColour">
      <property name="text">
       <string>Select Colour</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ContactBuilder</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ContactBuilder</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
